policyengine_us/variables/gov/usda/snap/snap_min_allotment.py

```diff
@@ -34,4 +34,23 @@ def formula(spm_unit, period, parameters):
             in_dc = state_code == "DC"
             min_allotment = where(in_dc, dc_min_allotment, min_allotment)
 
+        p_nm = parameters(period).gov.states.nm.snap.min_allotment
```

Comment:
move the 
```
state_code = spm_unit.household("state_code_str", period)
```
part to like 30 and remove from each of the if loops to avoid duplicating code